geoutils/georaster.py

```diff
@@ -484,7 +484,7 @@ def reproject(self, dst_ref=None, dst_crs=None, dst_size=None, dst_bounds=None,
                 raise ValueError("Either of `dst_ref` or `dst_crs` must be set. Not both.")
         else:
             if dst_crs is None:
-                raise ValueError("One of `dst_ref` or `dst_crs` must be set.")
+                dst_crs = self.crs
```

Comment:
I would raise a warning here: "destination crs not specified, not reprojecting"

Reply:
As said in the main discussion, there is still a case where dst_size (or res) can be set and it will require reshaping the array without changing the CRS. This is why I allow this.